docs/sma/plugin-pdf.md

```diff
@@ -191,4 +192,8 @@ A: Create an [issue ticket on github](https://github.com/supermemo/SuperMemoAssi
 
 **Q: How to I jump to a specific page number ?**
 
-A: Use <kbd>Ctrl</kbd> + <kbd>G</kbd>.
\ No newline at end of file
+A: Use <kbd>Ctrl</kbd> + <kbd>G</kbd>.
+
+**Q:How do I disable the Windows Sound from playing when I extract from a PDF?**
+
+A: Currently there isn't any workaround, you can disable system sounds by right clicking on the volume icon on your taskbar and setting it to zero.  
```

Comment:
I would argue that this is a workaround, since it effectively cancel the issue by another mean that a feature of SMA.